packages/code-du-travail-frontend/src/common/__tests__/ErrorBoundary.test.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { render } from "react-testing-library";
+import { ErrorBoundary } from "../ErrorBoundary";
+
+describe("Error Boundary", () => {
+  // Hack(lionelb): remove noisy error logging catch by ErrorBoundary
+  // @see https://github.com/facebook/react/issues/11098#issuecomment-412682721
+  const topLevelErrors = [];
+  function handleTopLevelError(event) {
+    // calling preventDefault() will prevent error logging
+    event.preventDefault();
+    topLevelErrors.push(event.error);
+  }
+
+  beforeEach(() => {
+    window.addEventListener("error", handleTopLevelError);
```

Comment:
Je me demande si il ne faut pas vider topLevelError dans le beforeEach ? Enfin là ça passe clairement mais si on rajoute d'autres "it" ?